tests/tas-elasticsearch/src/test/java/org/alfresco/elasticsearch/AlfrescoStackInitializer.java

```diff
@@ -297,4 +282,115 @@ protected GenericContainer createAlfrescoContainer()
                         "/usr/local/tomcat/webapps/alfresco/WEB-INF/classes/alfresco/search/elasticsearch/config/exactTermSearch.properties",
                         BindMode.READ_ONLY);
     }
+
```

Comment:
I kinda wonder if at this point it wouldn't be better to extract those into another class and provide them through configuration? Up to you, but it seems like a waste to separate it so neatly, just to not make it configurable (if different configs would ever be used, perhaps this is as static as they come)

Reply:
I would leave it as it is. It's not needed for/related to the refactoring.

Reply:
After looking through code I feel like I know it a bit better, I understand that it kinda needs to be static anyway and passing it through configuration doesn't make much sense, but I still think it's separate enough to warrant being put in another file, even if only reason would be making the code less cluttered.